controlplane/pkg/apis/plugins/registry_plugin.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package plugins;
+
+import "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/registry/registry.proto";
+import "ptypes/empty/empty.proto";
+
+message NSEList {
+    repeated registry.NetworkServiceEndpoint network_service_endpoints = 1;
+}
+
+service RegistryPlugin {
+    rpc RegisterNSM (registry.NetworkServiceManager) returns (registry.NetworkServiceManager);
+    rpc RegisterNSE (registry.NSERegistration) returns (registry.NSERegistration);
+    rpc RemoveNSE (registry.RemoveNSERequest) returns (google.protobuf.Empty);
+    rpc GetNSEs (google.protobuf.Empty) returns (NSEList);
+}
```

Comment:
Why are we creating a duplicate Registry api in addition to the existing one?

Reply:
Same for Registry API. In this PR I only add the new model. On the next PR I will remove the old model. So we will have only one.

Reply:
It is planned to leave only NetworkServiceRegistry service since it is used by NSEs. NetworkServiceDiscovery and NsmRegistry services will be replaced with the plugins